lib/percona_migrator/log_sanitizers/connection_details_sanitizer.rb

```diff
@@ -0,0 +1,19 @@
+module PerconaMigrator
+  module LogSanitizers
+    class ConnectionDetailsSanitizer
+
+      delegate :password_argument, to: :connection_details
+
+      def initialize(connection_details)
+        @connection_details = connection_details
+      end
+
+      def execute(log_statement)
+        password_argument.blank? ? log_statement : log_statement.gsub(connection_details.password_argument, '[filtered_password]')
```

Comment:
Could you use an early return for this?

Something like this would work

```ruby
return log_statement if password_argument.blank?

log_statement.gsub(password_argument, '[filtered_password]')
```

It'd also be great if you could store that `'[filtered_password]'` on a
constant.